Extensions/Settings/Dnn.PersonaBar.SiteSettings/Services/Dto/UpdateOtherSettingsRequest.cs

```diff
@@ -28,8 +28,5 @@ namespace Dnn.PersonaBar.SiteSettings.Services.Dto
 {
     public class UpdateOtherSettingsRequest
```

Comment:
Should this class be removed completely?

Reply:
I hesitated. Because we may (within the next few weeks) discover settings that should find their way here. Can we continue with this PR and decide this in the run-up to the 9.3 release?